src/style.module.css

```diff
@@ -342,47 +342,43 @@
   }
 }
 
-.weeks_next_enter {
-  animation: rdp-slide_in_right var(--rdp-animation_duration)
-    var(--rdp-animation_timing) forwards;
+.weeks_prev_enter {
+  animation: rdp-weeks_prev_enter var(--rdp-animation_duration)
+    var(--rdp-animation_timing);
 }
-
 .weeks_prev_exit {
-  animation: rdp-slide_out_left var(--rdp-animation_duration)
-    var(--rdp-animation_timing) forwards;
+  animation: rdp-weeks_prev_exit var(--rdp-animation_duration)
+    var(--rdp-animation_timing);
 }
 
-.weeks_prev_enter {
-  animation: rdp-slide_in_left var(--rdp-animation_duration)
-    var(--rdp-animation_timing) forwards;
+.weeks_next_enter {
+  animation: rdp-weeks_next_enter var(--rdp-animation_duration)
+    var(--rdp-animation_timing);
 }
 
 .weeks_next_exit {
-  animation: rdp-slide_out_right var(--rdp-animation_duration)
-    var(--rdp-animation_timing) forwards;
-}
-
-[dir="rtl"] .weeks_next_enter {
-  animation: rdp-slide_in_left var(--rdp-animation_duration)
-    var(--rdp-animation_timing) forwards;
-}
-
-[dir="rtl"] .weeks_prev_exit {
-  animation: rdp-slide_out_right var(--rdp-animation_duration)
-    var(--rdp-animation_timing) forwards;
+  animation: rdp-weeks_next_exit var(--rdp-animation_duration)
+    var(--rdp-animation_timing);
 }
 
-[dir="rtl"] .weeks_prev_enter {
-  animation: rdp-slide_in_right var(--rdp-animation_duration)
-    var(--rdp-animation_timing) forwards;
+@keyframes rdp_caption_prev_enter {
+  from {
+    opacity: 0;
+  }
+  to {
+    opacity: 1;
+  }
 }
-
-[dir="rtl"] .weeks_next_exit {
-  animation: rdp-slide_out_left var(--rdp-animation_duration)
-    var(--rdp-animation_timing) forwards;
+@keyframes rdp_caption_prev_exit {
+  from {
+    opacity: 1;
+  }
+  to {
+    opacity: 0;
+  }
 }
 
-@keyframes rdp-fade_in {
```

Comment:
The same as the comment before, `fade_in` is used in 2 classes, `caption_next_enter` and `caption_prev_enter`, so it's more confusing naming it `caption_next_enter` than what it represents visually, which is a `fade_in` animation.